gapic/utils/options.py

```diff
@@ -134,6 +138,7 @@ def tweak_path(p):
             lazy_import=bool(opts.pop('lazy-import', False)),
             old_naming=bool(opts.pop('old-naming', False)),
             add_iam_methods=bool(opts.pop('add-iam-methods', False)),
+            transport=opts.pop('transport', ['grpc'])[0].split('+')
```

Comment:
Can you add a quick comment indicating the expected format of the 'transport' option and maybe an example?